Makefile

```diff
@@ -219,6 +219,13 @@ deploy: clean-helm-rendered
 	helm template scanner chart/ --set tag=$(TAG),logLevel=$(LOGLEVEL),updateInterval=2m --output-dir rendered-chart
 	kubectl apply -R -f rendered-chart
 
+.PHONY: deploy-slim
+deploy-slim: clean-helm-rendered
+	@echo "+ $@"
+	kubectl create namespace stackrox || true
```

Comment:
Do we ignore failures here because the namespace might already exist, or is it something else?

Reply:
Yeah, in case it already exists